documentation/assertions/Promise/to-be-fulfilled.md

```diff
@@ -46,13 +44,13 @@ a regular expression, a function, or an object:
 
 
 ```javascript#async:true
```

Comment:
Yes that is what I'm proposing, but that can be done later. This is a very nice improvement!